pkgdown/versions.yaml

```diff
@@ -0,0 +1,27 @@
+- ref: 'refs/remotes/origin/main'
+  href: /dev
+  label: true
+- ref: 'refs/tags/v1.7.0'
```

Comment:
It's uncommon, but we sometimes release patch versions (e.g. v1.2.1) - what about them? I see a couple of options:
1. Display all versions.
2. Only display the latest patch version for each minor version.
3. Option (2), but hide the patch in label (e.g. use 1.2.1 for ref and href, but just 1.2 for label).

Personally I'd go with (3).

Reply:
I think this would be addressed in the release process. If you release a patch version, then update the ref accordingly.

I do agree that using two-part version makes more sense. However, this requires adding version detection. I thought about it, but initially decided against it as we have quite a bit of different tag formats.

Reply:
My message wasn't quite clear; I had in mind simple changes to `versions.yml`, e.g. show only 1.2 in the version selector, but link to 1.2.1 (the latest patch version for 1.2). See [this commit](https://github.com/Appsilon/rhino/pull/575/commits/fd5a6566ffb0850ae117cd8b01161f5010687bb0).

Reply:
That makes so much more sense now! 😁